libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_DNS.c

```diff
@@ -922,7 +922,7 @@ size_t xChunkLength;
 		return pdFAIL;
 	}
 	/*-----------------------------------------------------------*/
-#endif /* if ( ipconfigDNS_USE_CALLBACKS == 1 ) */
+#endif /* if ( ipconfigDNS_USE_CALLBACKS == 1 ) || ( ipconfigUSE_LLMNR == 1 ) */
 
```

Comment:
Asynchronous look-ups that use LLMNR also need the function `ulDNSHandlePacket()`, to receive asynchronous replies.